R/header.R

```diff
@@ -0,0 +1,36 @@
+#' DfE header banner
+#'
+#' @description
+#' This function uses `shinyGovstyle::header()` to create a header banner
+#' using the DfE logo.
+#'
+#' @param secondary_text Secondary header to supplement the main text
+#' @param secondary_link Add a link for clicking on secondary header
+#' @return a header html shiny object
+#'
+#' @seealso [shinyGovstyle::header()]
+#' @export
+#' @examples
+#'
+#' if (interactive()) {
+#'   ui <- fluidPage(
+#'     dfeshiny::header(
+#'       secondary_text = "User Examples"
+#'     )
+#'   )
+#'
+#'   server <- function(input, output, session) {}
+#'
+#'   shinyApp(ui = ui, server = server)
+#' }
+header <- function(secondary_text,
+                   secondary_link = "#") {
+  shinyGovstyle::header(
```

Comment:
Something I'd like to do here, which adds to the scope a little, but I think is manageable, is to tackle the accessibility issues with the shinygovstyle header function. It's been bugging me for ages that the main link and image fail accessibility tests on the following counts:

- the link doesn't have any text
- the image doesn't have any alt text

I feel like this is a good opportunity to fix that, so what I'd propose is to create a sub-function in this file called `shinygovstyle_header_accessible()`, copy the code as is from shinygovstyle header and then edit it to remove the accessibility fails and then call shinygovstyle_header_accessible on this line here instead of shinyGovstyle::header().

By keeping it as a sub-function, we set it up so that it's easy to port into shinyGovstyle once we've tested it out and we're happy to move it across. At that point we can remove the sub function here and revert to `shinyGovstyle::header()`.

Example code I found that should be a good starting point for fixing the accessibility issues:

 <a href="http://theblablasite.com/" target="_blank"
     aria-label="Visit The Blabla Site Dot Com (opens in a new window)">
       <img src="logo_or_whatever.png" alt="The Blabla site" />
</a>